cloud_experts_tutorials/cloud-experts-dynamic-certificate-custom-domain.adoc

```diff
@@ -17,17 +17,22 @@ toc::[]
 //   Kevin Collins
 //---
 
-While wildcard certificates provide simplicity by securing all first-level subdomains of a given domain with a single certificate, other use cases may require the use of individual certificates per domain. This tutorial guides you through using the link:https://docs.openshift.com/container-platform/latest/security/cert_manager_operator/index.html[cert-manager Operator for Red Hat OpenShift] and link:https://letsencrypt.org/[Let's Encrypt] to dynamically issue certificates for routes created using a custom domain.
+While wildcard certificates provide simplicity by securing all first-level subdomains of a given domain with a single certificate, other use cases can require the use of individual certificates per domain.
 
+Learn how to use the link:https://docs.openshift.com/container-platform/latest/security/cert_manager_operator/index.html[cert-manager Operator for Red Hat OpenShift] and link:https://letsencrypt.org/[Let's Encrypt] to dynamically issue certificates for routes created using a custom domain.
+
+[id="cloud-experts-dynamic-certificate-custom-domain-prerequisites"]
 == Prerequisites
 
 * A ROSA cluster
-* You have access to the OpenShift CLI (`oc`)
-* You have access to the AWS CLI (`aws`)
+* A user account with `cluster-admin` privileges
+* The OpenShift CLI (`oc`)
+* The Amazon Web Services (AWS) CLI (`aws`)
 * A unique domain, such as *.apps.<company_name>.io
 * An Amazon Route 53 public hosted zone for the above domain
 
-=== Setting up your environment
+[id="cloud-experts-dynamic-certificate-custom-domain-environment-setup"]
+== Setting up your environment
 
 . Configure the following environment variables:
 +
```

Comment:
Should the value for AWS_PAGER be blank?

Reply:
This value is set to blank in all MOBB tutorials that include this same setup section.